src/feature-test/user_see_form.test.js

```diff
@@ -1,10 +1,35 @@
-import '../setupTests.js'
-import App from '../App';
+import '../setupTest.js'
 
-it('renders without crashing', () => {
-  const wrapper = mount(<App />);
-  const input = wrapper.find('#input1')
-  input.simulate("change", { target: { value: "potatoes" }})
-  // expect(wrapper.text()).toEqual("potatoes");
+
+describe('I.I.A.S', () => {
+  it('the user can upload recipes inputing ingredients', async () => {
+    const browser = await puppeteer.launch({
+      slowMo: 40,
+      // headless: false,
+      // devtools: true
+    });
+    const page = await browser.newPage();
+    await page.goto('http://localhost:3000/');
+    await page.type('input[name="ing1"]', 'potatoes');
+    await page.type('input[name="ing2"]', 'sugar');
+    await page.type('input[name="ing3"]', 'milk');
+    await page.setRequestInterception(true);
+    await page.on('request', request => {
+      if(request.resourceType(fetch)) {
+        request.respond({
+          body: JSON.stringify(
+            [{id: 6, title: "Champ (Mashed Potatoes from Ulster)", image: "https://spoonacular.com/recipeImages/697254-312x231.jpg"}]        
+          )
+        })
+      } else {
```

Comment:
What is this else doing? Is it needed? Currently it doesn't appear do bring any value. Am I missing something?

Reply:
deleted